docker-compose.yml

```diff
@@ -9,9 +9,11 @@ services:
     environment:
       REDIS_HOST: redis
       REDIS_PORT: 6379
+      FORCE_HTTPS: disabled
       JWT_KEY: "uZrJ!xe*xN?!;oU.u*;QOSM+|=4C?WH?6eWPcK/6AkIXIVGQguSA*r"
     ports:
```

Comment:
I think we don't need to actually map container's ports to the host machine's ports
Means replace `3000:3000` with just `3000` and sane for 4000 port